embark-theme.conf

```diff
@@ -0,0 +1,53 @@
+# Embark (https://github.com/embark-theme/kitty) {{{
+background #1e1c31
+foreground #EEFFFF
+cursor #A1EFD3
+selection_background  #fbfcfc
+
+# black
+color0 #1e1c31
+color8  #585273
+
+# red
+color1 #F0719B
+color9 #F02E6E
+
+# green
+color2 #A1EFD3
+color10 #2CE592
+
+# yellow
+color3       #ffe9aa
+color11      #ffb378
+
+# blue
+color4 #57C7FF
+color12 #1DA0E2
+
+# magenta
+color5 #C792EA
+color13 #A742EA
+
+# cyan
+color6       #87DFEB
+color14      #63f2f1
+
+# white
+color7 #F8F8F2
+color15 #a6b3cc
+
+
+active_border_color #A1EFD3
+inactive_border_color #a6b3cc
+bell_border_color #f56574
+
+active_tab_foreground   #222433 
+active_tab_background   #63f2f1
+active_tab_font_style   bold
+
+inactive_tab_foreground #63f2f1
+inactive_tab_background #222433 
```

Comment:
Small tweak I would like to ask for here just to keep this as close to the vim status line as possible. For the inactive tab bar styles what do you think about this:
```
inactive_tab_foreground #a6b3cc
inactive_tab_background #585273
```

Here is how it looks:
<img width="782" alt="Screen Shot 2020-08-28 at 1 09 29 PM" src="https://user-images.githubusercontent.com/13260877/91606721-d8e76700-e92f-11ea-8fc7-e30651f12377.png">

What are your thoughts?

Reply:
Sounds good, amending rn.